test/integration/cni/service_connectivity_test.go

```diff
@@ -57,6 +59,11 @@ var _ = Describe("[CANARY] test service connectivity", func() {
 	var negativeTesterContainer v1.Container
 
 	JustBeforeEach(func() {
+		k8sUtils.AddEnvVarToDaemonSetAndWaitTillUpdated(f, "aws-node", "kube-system",
```

Comment:
Let's add a comment here that we are setting IP cooldown to 5 seconds for each test case. Also, let's use `utils.AwsNodeName` and `utils.AwsNodeNamespace` here